server/routes/api/v1/dbinsert.js

```diff
@@ -0,0 +1,61 @@
+const { Client } = require('pg');
+const config = require('config');
+const client = new Client({
+  user: config.get('userName'),
+  password: config.get('myPass'),
+  port: 5432,
+  database: 'test'
+});
+const connectDB = async () => {
+  await client.connect();
+  console.log('Connected successfully');
+};
+
+const insertUser = async (name, email, avatar, password) => {
+  const table = 'users';
+  try {
+    await client.query('BEGIN');
```

Comment:
I noticed that you have been doing your queries in `TRANSACTIONs`. Transactions should be used when you want to execute dependent and sequential queries(ie one should not run if the other fails). 

You do not need it when exectuing only one query

Reply:
Any update to this?